crates/core/src/kernel/models/schema.rs

```diff
@@ -641,6 +643,7 @@ mod tests {
     use super::*;
     use serde_json;
     use serde_json::json;
+    use std::hash::DefaultHasher;
```

Comment:
incase anyone else runs into this - this raises the MSRV of running the tests to 1.76.

Reply:
thanks for pointing this out!